{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -44,7 +44,11 @@ jobs:
         - docker pull {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${IMAGE_TAG}
       deploy:
         provider: script
-        script: ./scripts/push_to_hub.sh
+        script:
+         - docker tag ${IMAGE_TAG} {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${TRAVIS_COMMIT::12}
```

Comment:
I think the first part needs to be `docker tag {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${IMAGE_TAG} ...` or is this some kind of shortcut?